src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -750,7 +751,7 @@ private static bool TryDeserializeJson<TResult>(string json, out TResult result)
 
             try
             {
-                result = JsonConvert.DeserializeObject<TResult>(json);
+                result = JsonConvert.DeserializeObject<TResult>(json, _datetimeSerializerSettings);
```

Comment:
This would be a breaking change, right? We can't make a breaking change.

Reply:
Argh, yeah you're right. Thanks for the call-out. Fixed in V2, so I'll abandon this PR.